docs/logging/logging.md

```diff
@@ -1,24 +1,46 @@
 ---
 sidebar_position: 1
-sidebar_label: Logging
+sidebar_label: Logging Audit Event
 title: ""
 keywords:
 - Harvester
-- harvester
 - Logging
+- Audit
+- Event
 ---
 
-# Harvester Logging
+# Harvester Logging Audit Event
 
 _Available as of v1.1.0_
 
-The Harvester logging infrastructure allows you to aggregate Harvester logs into an external service such as [Elasticsearch](https://www.elastic.co/elasticsearch/),
-[Grafana Loki](https://grafana.com/oss/loki/), or [Splunk](https://www.splunk.com/).
+It is important to know what is happending / has happend in the `Harvester Cluster`.
 
-## Collected Logs
+`Harvester` collects the `cluster running log`, kubernetes `audit` and `event` log right after the cluster is powered on, which is helpful for monitoring, logging, auditing and troubleshooting.
+
+`Harvester` supports to send those logs to vairous types of log server.
+
+:::note
+The logging related data size is related on the cluster scale, workload and other factors. `Harvester` does not use persistant storage to store log data inside the cluster. Users need to set up log server to receive logs accordingly.
```

Comment:
```suggestion
The logging related data size is related on the cluster scale, workload and other factors. `Harvester` does not use persistant storage to store log data inside the cluster. Users need to set up a log server to receive logs accordingly.
```

Reply:
consider refactoring to:

The size of data associated with logs is related to cluster size, workload, and other factors. `Harvester` does not use persistent storage to store log data within the cluster. Users need to set up a log server to receive the corresponding logs.